### PR TITLE
Fix failing NVQC test

### PR DESCRIPTION
**Pull request by annagrin**
### Description
Disable NVQC test that runs old argument synthesis until the new argument synthesis is plugged in into NVQC as well.

See https://github.com/NVIDIA/cuda-quantum/issues/2146
Task to enable the test is listed in https://github.com/NVIDIA/cuda-quantum/issues/1931

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -553,7 +553,8 @@ jobs:
           for filename in `find targettests/Remote-Sim -name '*.cpp'`; do
             # unsupport_args and compile_errors are compile error tests
             # nested_vectors: Compiler fails to synthesize nested vector parameters (https://github.com/NVIDIA/cuda-quantum/issues/2001)
-            if [[ "$filename" != *"unsupport_args"* ]] && [[ "$filename" != *"compile_errors"* ]] && [[ "$filename" != *"nested_vectors"* ]]; then
+            # state_init: New argument synthesis is not executed for nvqc (https://github.com/NVIDIA/cuda-quantum/issues/2146)
+            if [[ "$filename" != *"unsupport_args"* ]] && [[ "$filename" != *"compile_errors"* ]] && [[ "$filename" != *"nested_vectors"* ]] && [[ "$filename" != *"state_init"* ]]; then
```
Comment (bmhowe23) [inline .github/workflows/integration_tests.yml:557]: Wouldn't this also disable targettests/Remote-Sim/state_init_vector.cpp, too? Is that necessary/desired?

Comment (annagrin) [reply .github/workflows/integration_tests.yml:557]: yes, thanks! Renamed that test since the name was confusing.

Comment (annagrin) [reply .github/workflows/integration_tests.yml:557]: Now I am thinking both names are confusing. Will name them `qvector_init_vector.cpp` and `qvector_init_state.cpp` to indicate that we are initializing a `qvector` from vector or state arguments.

Comment (annagrin) [reply .github/workflows/integration_tests.yml]: done!

**Review by bmhowe23 — APPROVED** [review]
Thanks, @annagrin

**Pull request merged**